### ISSUE TITLE
Accessibility (a11y) bug in Sign in modal

### ISSUE THREAD

**Issue by asg5704** [department-of-veterans-affairs/va.gov-team#54607]
## Description
There is currently an accessibility bug that exists in the Sign in Modal that should be fixed

## Tasks
- [ ] Use the aXe browser extension to run the test suite on the Sign in Modal
- [ ] Fix the outlying issue for the Sign in Modal
